src/styles.css

```diff
@@ -0,0 +1,1195 @@
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 50px;
+  padding: -1%;
+  background-color: #fff;
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  overflow: hidden;
+}
+
+.header-container #myLinks {
+  display: none;
+}
+
+.nav-item1 {
+  flex-grow: 1;
+  transition: all 1s;
+}
+
+.nav-item1:hover {
+  transform: rotate(360deg);
+}
+
+.nav-item2 {
+  flex-grow: 8;
+}
+
+.nav-item3 {
+  flex-grow: 1;
+}
+
+.overlay {
+  height: 100%;
+  width: 100%;
+  display: none;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: #6070ff;
+  opacity: 0.95;
+}
+
+.overlay-content {
+  position: relative;
+  top: 5%;
+  width: 100%;
+  text-align: left;
+  margin-top: 30px;
+  padding-left: 26px;
+}
+
+.hamburger-span {
+  color: #6070ff;
+  font-size: 30px;
+  cursor: pointer;
+}
+
+.overlay a {
+  padding: 8px;
+  text-decoration: none;
+  font-size: 32px;
+  color: #fff;
+  font-weight: 600;
+  display: block;
+  transition: 0.3s;
+  line-height: 44px;
+}
+
+.overlay a:hover,
+.overlay a:focus {
+  color: #f1f1f1;
+}
+
+.overlay .closebtn {
+  position: absolute;
+  top: 11px;
+  right: 133px;
+  font-size: 30px;
+  font-weight: normal;
+  border: 2px;
+  cursor: pointer;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  background-image: url("../assets/header.png");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-color: #fff;
+  width: 100%;
+  height: 90vh;
+  border-radius: 0 0 0 100px;
+  border-bottom: 2px solid #cfd8dc;
+}
+
+.intro {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  width: 70%;
+  margin: 5%;
+  margin-top: 25%;
+  animation-name: introin;
+  animation-duration: 4s;
+}
+
+@keyframes introin {
+  from {
+    margin-left: 100%;
+  }
+
+  to {
+    margin-left: 22%;
+  }
+}
+
+textarea::placeholder {
+  color: #b3bac5;
+  padding-top: 15px;
+  font-size: 15px;
+}
+
+input[type="name"]::placeholder {
+  color: #172b4d;
+  font-size: 17px;
+}
+
+input[type="email"]::placeholder {
+  color: #172b4d;
+  font-size: 17px;
+}
+
+.intro1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  position: static;
+  width: 90%;
+  height: 122px;
+  left: 24px;
+  top: 276px;
+  color: #344563;
+  margin-top: 0;
+  margin-top: -40px;
+}
+
+p.intro1 {
+  padding: 15px;
+}
+
+.intro2 {
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  align-items: flex-start;
+  padding-left: 15px;
+  padding-top: 35px;
+  grid-gap: 30px;
+  justify-items: center;
+  margin-left: 20%;
+}
```

Comment:
## [Required changes]
- The position of the subtitle "Let's connect" is currently not the same as the Figma design. Kindly fix it

Figma Design
![image](https://user-images.githubusercontent.com/89091822/235268152-9fec127f-ccd2-46b1-8cdf-6b310f4e46e3.png)


Your Design
![image](https://user-images.githubusercontent.com/89091822/235268067-fad24f8d-87b4-4a06-a036-8503c44cb17a.png)